src/main/java/com/epam/ta/reportportal/core/configs/SpringDocConfiguration.java

```diff
@@ -129,6 +135,37 @@ public OpenApiCustomiser sortSchemasAlphabetically() {
     };
   }
 
+  /**
+   * Resolve Iterable schema responses
+   * @return ModelConverter
+   */
+  @Bean
+  public ModelConverter iterableModelConverter() {
+    return new ModelConverter() {
+      @Override
+      public Schema resolve(AnnotatedType annotatedType, ModelConverterContext context,
+          Iterator<ModelConverter> chain) {
+        JavaType javaType = Json.mapper().constructType(annotatedType.getType());
+        if (javaType != null) {
```

Comment:
is it possible to combine two `if` blocks into single one?

Reply:
done